recipes/nng/all/conanfile.py

```diff
@@ -0,0 +1,88 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from glob import glob
+import os
+
+
+class NngConan(ConanFile):
+    name = "nng"
+    description = "nanomsg-next-generation: light-weight brokerless messaging"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "src/*.cmake", "src/*.txt"]
+    homepage = "https://github.com/nanomsg/nng"
+    license = "MIT"
+    topics = ("nanomsg", "communication", "messaging", "protocols")
+    generators = "cmake"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "tests": [True, False],
+        "nngcat": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "tests": False,
```

Comment:
I think *tests* can be removed. The idea is packaging the library, not running the unit tests. It can useful when conan is used as in-source (same repository for sources and conan recipe), and conan triggers cmake build to the project, but as external recipe I see it as useless.

Reply:
I agree. I'll remove the option an explicitly disable building and running the tests (it's enabled by default, except when cross-compiling).